scripts/kitchen_test.sh

```diff
@@ -87,15 +87,13 @@ if [ ${#failures_array[@]} -eq 0 ]; then
   dos2unix test/results/*
   read -p 'Commit test results and push to github? [y]' -n 1 -r REPLY
   echo
-  commit_message="test: on premise windows 10 and windows server \'kitchen test\' results"
+  commit_message="test: on premise windows 10 and windows server `kitchen test` results"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/gh/clearasmudd/windows-formula/pullRequest?prid=5177273)